packages/angular/src/generators/component/lib/module.ts

```diff
@@ -80,5 +80,7 @@ function findModule(
     dir = dirname(dir);
   }
 
-  return null;
+  throw new Error(
+    "Could not find a candidate module to add the component to. Please specify which module the component should be added to by using the '--module' option."
+  );
```

Comment:
@Coly010 do you think we should tune this to also mention `--standalone`? Something like: 

```suggestion
  throw new Error(
    "Could not find a candidate module to add the component to. Please specify which module the component should be added to by using the '--module' option, or pass '--standalone' to use Angular's standalone API"
  );
```

Reply:
yes that might make sense